cli/command/cli_test.go

```diff
@@ -66,6 +66,7 @@ func TestNewAPIClientFromFlagsForDefaultSchema(t *testing.T) {
 func TestNewAPIClientFromFlagsWithAPIVersionFromEnv(t *testing.T) {
 	customVersion := "v3.3.3"
 	defer env.Patch(t, "DOCKER_API_VERSION", customVersion)()
+	defer env.Patch(t, "DOCKER_HOST", ":2375")()
```

Comment:
nit: shouldn't this be in another test (subtest ? :upside_down_face:)

Reply:
This is required to make sure the initializer goes trough the "don't use context store" code path. (I consider those tests as non-regression tests for existing scripts)